src/styles/utils/generators/ModalStyleUtils.ts

```diff
@@ -197,7 +197,7 @@ const createModalStyleUtils: StyleUtilGenerator<GetModalStylesStyleUtil> = ({the
                     },
                 };
                 modalContainerStyle = {
-                    borderRadius: 12,
+                    borderRadius: 16,
```

Comment:
Is this value stored as a variable anywhere?

Reply:
Completely agree. We should avoid hardcoding values. 
Isn’t `componentBorderRadiusLarge` the variable here?

Reply:
I believe so! https://github.com/Expensify/App/blob/350b7ab89164db9eac714be79bcd3470ca756aee/src/styles/variables.ts#L28